src/qml/main.qml

```diff
@@ -5,6 +5,52 @@ import harbour.kaidan 1.0
 Kirigami.ApplicationWindow {
 	id: root
 	visible: true
+	
+	globalDrawer: Kirigami.GlobalDrawer {
+		title: "Kaidan"
+		titleIcon: "kopete"
+		bannerImageSource: "qrc:/banner.png"
+
+		actions: [
+		Kirigami.Action {
+			text: "Status"
+			iconName: "user-status-pending-symblolic"
+			Kirigami.Action {
+				text: "Online"
+				onTriggered: showPassiveNotification("coming soon")
+				iconName: "im-user-online"
+			}
+			Kirigami.Action {
+				text: "Offline"
+				onTriggered: showPassiveNotification("coming soon")
+				iconName: "im-user-offline"
+			}
+			Kirigami.Action {
+				text: "AFK"
+				onTriggered: showPassiveNotification("coming soon")
+				iconName: "im-user-away"
+			}
+		},
+		Kirigami.Action {
+			text: "My Profile"
+			iconName: "view-list-icons"
+		},
+		Kirigami.Action {
+			text: "Settings"
+			iconName: "menu_new"
+			Kirigami.Action {
+				text: "login"
+				iconName: "system-switch-user"
+				onTriggered: applicationWindow().pageStack.replace(loginPageComponent)
```

Comment:
This is active even after user is logged in. I think this should become inactive after login.